pandas/core/config.py

```diff
@@ -388,12 +390,14 @@ class option_context(object):
 
     """
 
-    def __init__(self, *args):
-        if not (len(args) % 2 == 0 and len(args) >= 2):
+    def __init__(self, *args, options={}):
+        if not (len(args) % 2 == 0 and (len(args) >= 2 or len(options))):
```

Comment:
I interpret this as ensuring that there are an even amount of varargs. Not sure an existence check of options belongs as part of this conditional? If so though would want to update the ValueError to clarify the role of `options`